src/components/data-exploration/heatmap/HeatmapPlot.jsx

```diff
@@ -128,9 +138,10 @@ const HeatmapPlot = (props) => {
   }, [selectedGenes, loadingGenes, markerGenesLoading]);
 
   useConditionalEffect(() => {
-    if (!selectedGenes?.length > 0
-      || cellSets.hierarchy.length === 0
-      || downsampledCellOrder?.length === 0
+    if (
+      !selectedGenes?.length > 0
+      || !cellSets.hierarchy.length > 0
```

Comment:
just !cellSets.hierarchy.length  works too right?

Reply:
y, but the thing is that just doing `cellSets.hierarchy.length` is less specific about what you are checking for